alpine/3.14/7.4/Dockerfile

```diff
@@ -98,7 +99,7 @@ RUN apk update \
     && \
     pecl install -o -f ${PHP_PECL_EXTENSIONS} xdebug \
     && rm -rf /tmp/pear \
-    && docker-php-ext-enable ${PHP_EXTENSIONS} ${PHP_PECL_EXTENSIONS} \
+    && docker-php-ext-enable ${PHP_EXTENSIONS} redis \
```

Comment:
Let's introduce a new variable for additional extensions for xdebug + hprof and revert the current changes.